src/librustc/diagnostics.rs

```diff
@@ -1939,6 +1939,65 @@ fn foo<'a>(x: &'a i32, y: &i32) -> &'a i32 {
 ```
 "##,
 
+E0638: r##"
```

Comment:
You need to add this to the diagnostics file of librustc_typeck

Reply:
Thanks, I've pushed up a fix for this. I wasn't aware that there was a diagnostics file in the other crates, my bad.